src/main/java/aquality/selenium/core/elements/interfaces/IParent.java

```diff
@@ -1,8 +1,11 @@
 package aquality.selenium.core.elements.interfaces;
 
 import aquality.selenium.core.elements.ElementState;
+import aquality.selenium.core.elements.ElementsCount;
 import org.openqa.selenium.By;
 
+import java.util.List;
+
 public interface IParent {
```

Comment:
IElement extends IParent? (I do not see that Element implements IParent explicitly)

Reply:
yes it does